spec/requests/api/v1/reservations_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'swagger_helper'
+
+RSpec.describe 'api/v1/reservations', type: :request do
+  reservation_properties = {
+    city: { type: :string, default: 'Mexico City' },
+    starting_date: { type: :string, default: '07-04-2023' },
+    ending_date: { type: :string, default: '12-04-2023' },
+    car_id: { type: :number, default: '1' }
+  }
+  path '/api/v1/reservations' do
+    get('List all reservations for the current user') do
+      tags 'Reservations'
+      response(200, 'successful') do
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+        run_test!
+      end
+    end
+    post('Add a new reservation') do
+      tags 'Reservations'
+      response(200, 'successful') do
```

Comment:
In this line, I think the response should be documented as 201, not 200. Please see the screenshot.

![Screen Shot 2023-04-10 at 11 14 16 AM (2)](https://user-images.githubusercontent.com/101260188/230838948-5bc204ad-16ae-4ccb-b6b6-6690a08923ed.png)